kolibri/utils/server.py

```diff
@@ -319,11 +319,13 @@ def configure_http_server(port):
         conf.OPTIONS["Deployment"]["ZIP_CONTENT_PORT"],
     )
 
+    alt_port_app = wsgi.PathInfoDispatcher({"/": get_application(), CONTENT_ROOT: content_handler})
```

Comment:
Just need to run the code formatter bc of this long line

Reply:
Ugh, I guess pre-commit didn't run properly when I modified the commit.